app/src/hvr/cpp/native-lib.cpp

```diff
@@ -116,12 +123,15 @@ extern "C" {
 
 JNI_METHOD(void, nativeOnCreate)
 (JNIEnv *aEnv, jobject activity) {
+  LOGI("oncreate");
```

Comment:
Perhaps use `onCreate` (in camelCase) for consistency with the other similar logs.

Reply:
Sure